tests/test_blockchain.py

```diff
@@ -318,42 +336,46 @@ class TestReorgs:
             test_constants, 3, blocks, 9, b"3"
         )
         await b.receive_block(
```

Comment:
Not changed here, but this case is not being asserted. (it fails)

Reply:
Good catch thanks